docs/spark-writes.md

```diff
@@ -339,74 +331,61 @@ USING iceberg
 PARTITIONED BY (days(ts), category)
 ```
 
-To write data to the sample table, your data needs to be sorted by `days(ts), category`.
-
-If you're inserting data with SQL statement, you can use `ORDER BY` to achieve it, like below:
+To write data to the sample table, data needs to be sorted by `days(ts), category` but this is taken care
+of automatically by the default `hash` distribution. Previously this would have required manually sorting, but this 
+is no longer the case.
```

Comment:
when finishing this sentence, it's not clear what the below SQL example is trying to tell me. Maybe add a sentence saying that previously an `ORDER BY` was required in the below SQL

Reply:
The order by wasn't required before either, the "OrderBy" would automatically set the distirbution mode to range. I feel like that was just confusing.